pkg/templates/pod.tmpl

```diff
@@ -21,7 +21,7 @@ spec:
       hostPath:
         path: {{index .Options "dataPath"}}
   initContainers:
-  - image: volumecontroller/aws-cli
+  - image: minio/mc:RELEASE.2018-02-09T23-07-36Z
```

Comment:
Maybe we can get this in the chart?

Reply:
I'd want to tackle configurability of this in the code itself, and let people control this and the command via the CRD, then maybe make defaults usable in helm.

Or is this making it too generic?

Reply:
We should keep it this way I think, because we would end up with a PodSpec itself in the CR. And we should ideally control what goes in the command for a particular data source and not expose that externally.